ansible/roles/anitya-dev/tasks/main.yml

```diff
@@ -42,6 +42,7 @@
            fedora-messaging,
            python3-alembic,
            python3-arrow,
+           python-authlib,
```

Comment:
It's nice that the authlib is available in Fedora, one less package to download from PyPi.